README.md

```diff
@@ -1,7 +1,7 @@
 # Sagemaker-BMSG-GAN 
```

Comment:
Could you please let the title stay BMSG-GAN? I think it is not appropriate to mention AWS Sagemaker in the title, this being the accompanying code for the paper. The title about SageMaker Configuration Added is informative for the users to know that it is supported. Hope you don't mind. :smile: